SECURITY.md

```diff
@@ -30,13 +30,14 @@ Only critical severity bug fixes will be backported to past major releases.
 
 | Version | Critical security fixes | Other security fixes |
 | ------- | ----------------------- | -------------------- |
-| 4.x     | :white_check_mark:      | :white_check_mark:   |
+| 5.x     | :white_check_mark:      | :white_check_mark:   |
+| 4.9     | :white_check_mark:      | :x:                  |
```

Comment:
Do you agree with this? @Amxx
We probably would like to keep medium severity fixes in 4.9 along with the critical for some time.